Tests/scripts/hook_validations/script.py

```diff
@@ -80,7 +80,8 @@ def is_backward_compatible(self):
     def is_valid_script(self):
         """Check whether the Integration is valid or not, update the _is_valid field to determine that"""
         is_script_valid = any([
-            self.is_valid_subtype()
+            self.is_valid_subtype(),
+            self.is_valid_id()
```

Comment:
this is a problem since we currently have some script in that form.
this should be applied to only new files

Reply:
@yaakovi  after the fix of @GalRabin this should be fine right?

Reply:
should be fine now